config/checkstyle/sun_checks.xml

```diff
@@ -204,4 +204,12 @@
     <property name="severity" value="warning"/>
   </module>
 
+  <!-- Checks for Headers to not contain old license text -->
+  <module name="RegexpSingleline">
+    <property name="format" value="Portions Copyright OpenSearch Contributors"/>
```

Comment:
Looks like that regex needs a bit of improvment, here 
```suggestion
    <property name="format" value="\*( Portions)? Copyright OpenSearch Contributors"/>
```

With this change I found 5 errors in the following files:
```
[ant:checkstyle] [ERROR] /local/home/petern/git/security/src/main/java/org/opensearch/security/ssl/transport/DualModeSSLHandler.java:2: Incorrect license header [RegexpSingleline]
[ant:checkstyle] [ERROR] /local/home/petern/git/security/src/main/java/org/opensearch/security/ssl/transport/SSLConfig.java:2: Incorrect license header [RegexpSingleline]
[ant:checkstyle] [ERROR] /local/home/petern/git/security/src/main/java/org/opensearch/security/ssl/util/SSLConnectionTestUtil.java:2: Incorrect license header [RegexpSingleline]
[ant:checkstyle] [ERROR] /local/home/petern/git/security/src/main/java/org/opensearch/security/ssl/util/TLSUtil.java:2: Incorrect license header [RegexpSingleline]
[ant:checkstyle] [ERROR] /local/home/petern/git/security/src/main/java/org/opensearch/security/tools/SecurityAdmin.java:17: Incorrect license header [RegexpSingleline]
```

Reply:
good catch... thank you!